sql/sql_yacc.yy

```diff
@@ -5284,6 +5285,7 @@ tablespace_option:
         | ts_option_comment
         | ts_option_file_block_size
 	| ts_option_encryption
+        | ts_option_encryption_key_id
```

Comment:
Please fix ts_option_encryption indentation in all three chunks, not only the 1st one